packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -199,6 +209,10 @@ const WorkflowStepStore = types
           firstStep.taskKeys = firstTask.tasks
         }
 
+        if (firstTask.type === 'single' && !isThereBranching(firstTask)) {
```

Comment:
I don't love the `!isThereBranching` with the `!`, but consistent with the related code in Step? Open to suggestion.

Reply:
One time I miss coffeescript's style of `not isThereBranching firstTask` :smile:. I'm ok with this. It could be set to a variable for readability like `const isFirstSingleChoiceTaskNotBranching = firstTask.type === 'single' && !isThereBranching(firstTask)`. That's really verbose, but readable?

Reply:
I like the improved readability, there's a lot going on in this `convertWorkflowToUseSteps`, making it easier to understand++